nncf/common/graph/transformations/commands.py

```diff
@@ -211,3 +213,26 @@ def union(self, other: 'TransformationCommand') -> 'TransformationCommand':
 
     def __add__(self, other: 'TransformationCommand') -> 'TransformationCommand':
         return self.union(other)
+
+class Command:
+    """
+    The base class for non-target transformation commands.
+    """
+
+    def __init__(self, command_type: TransformationType):
+        """
+        Constructor.
+
+        :param command_type: Type of the non-target transformation command.
+        """
+        self._command_type = command_type
+
+    @property
+    def type(self) -> TransformationType:
+        return self._command_type
+
+    def union(self, other: 'TransformationCommand') -> 'TransformationCommand':
```

Comment:
Let's remove union and add methods from the Command class . It will simplify using Command.